packages/webapp/public/locales/en/translation.json

```diff
@@ -547,7 +511,6 @@
     "VALUE": "Value"
   },
   "EXPIRED_TOKEN": {
-    "RESET_PASSWORD": "This link has expired.",
```

Comment:
It's used, although I've worked on this flow and I _still_ don't how exactly.

<img width="375" alt="Screenshot 2023-11-10 at 6 10 43 PM" src="https://github.com/LiteFarmOrg/LiteFarm/assets/82857964/02cfc2e9-a54e-46f3-9378-60b99f194217">